Drive/rover_drive_system.hpp

```diff
@@ -141,7 +140,7 @@ class RoverDriveSystem : public sjsu::common::RoverSystem
   /// Checks if the rover got a new drive mode command
   bool IsNewMode()
   {
-    if (current_mode_ != mc_data_.drive_mode)
+    if (current_drive_mode_ != current_drive_mode_)
```

Comment:
This is a major bug!!! This is why we needed to have a mc drive mode variable to compare it to our current drive mode

Reply:
yea forgot about this